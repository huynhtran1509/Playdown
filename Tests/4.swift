/*: This is an optional comment
This should be the first line of output.
This is another line.
*/