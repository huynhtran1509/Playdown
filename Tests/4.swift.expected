This should be the first line of output.
This is another line.